mne/coreg.py

```diff
@@ -946,7 +946,7 @@ def scale_bem(subject_to, bem_name, subject_from=None, scale=None,
     surfs = read_bem_surfaces(src)
     for surf in surfs:
         surf['rr'] *= scale
-        if nn_scale is not None:
+        if nn_scale is not None and len(surf['nn']) > 0:
```

Comment:
Yeah I was thinking about reverting, but it should not hurt and would have prevented the issue @kingjr had in the first place